### DISCUSSION TITLE
[V2] Docs - Search not working

### DISCUSSION THREAD

**Discussion by francoism90** [spatie/laravel-data#170] [Q&A] [answered]
I don't know if you're aware, but the search engine isn't working on the V2 docs.

I've tried searching for collections, optional, etc. - but nothing happens.

Hopefully it can be fixed soon. :)

**Comment by rubenvanassche** [comment]
Hi @francoism90,

Algolia has updated their search service for open source software, we're trying to get us signed up again so that the search will start working again.

**Comment by HugoHeneault** [comment]
Hey there!
Any update about it? Search is soooo useful. Thanks :)

**Comment by rubenvanassche** [comment, marked as answer]
Took a bit longer than expected, but we have a completely new doc search on spatie.be!

**Reply by francoism90** [reply]
@rubenvanassche no Algolia anymore?

**Reply by rubenvanassche** [reply]
Nowp, Freek will be posting about this probably next week!

**Reply by francoism90** [reply]
@rubenvanassche Exciting, looking forward to it!
